sig/helpers.rbs

```diff
@@ -0,0 +1,35 @@
+module Helpers
+  # Convert an Hash to json
```

Comment:
I think we can remove all the comments in all the `.rbs` files (except the TODO items). It's just extra stuff to maintain, wdyt?

Reply:
completely agree